core/modules/field/modules/options/js/options.js

```diff
@@ -0,0 +1,849 @@
+
+/**
+ * @file
+ * Add JavaScript behaviors for the "options" form element type.
+ */
+
+(function($) {
+
+Backdrop.optionElements = Backdrop.optionElements || {};
+Backdrop.behaviors.optionsElement = Backdrop.behaviors.optionsElement || {};
+
+// We need to check/set HTML properties frequently, such as the disabled and
+// checked state of elements. In jQuery 1.6+, the "prop" method was added for
+// this purpose, but in earlier versions you had to use "attr".
+$.fn.oeProp = $.fn.prop ? $.fn.prop : $.fn.attr;
```

Comment:
We can now just assuming `$.fn.prop()`

Reply:
I noted several of these replacements, but there are dozens of them. A find/replace to just replace `oeProp` with `prop` would do it.

Reply:
done